tests/coded_tools/now_agents/integration_tests/test_integration_agent_discovery_debug.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+"""Test ServiceNow agents directly with more debugging"""
+
+import sys
+from pathlib import Path
+
+from dotenv import load_dotenv
+
+# Add the project root to Python path (need to go up 5 levels: integration_tests -> now_agents -> coded_tools -> tests -> project_root)
+project_root = Path(__file__).parent.parent.parent.parent.parent
+sys.path.insert(0, str(project_root))
+
+# Load environment
+env_path = project_root / ".env"
+load_dotenv(env_path)
+
+from coded_tools.now_agents.nowagent_api_get_agents import NowAgentAPIGetAgents
+
+
```

Comment:
We don't distinguish between unit and integration tests in this repo yet. We should. I'll probably have to temporarily:
```
@unittest.skip("No a unit test: this is an integration test that requires configuration.")
```
until we support them.
@donn-leaf heads up ^^^

Reply:
It breaks our automatic checks, yes. Can you either rename it to something else than `test_*`  e.g. `integration_test_*` (that should make `pytest` ignore it) 
Or disable it with a skip annotation like:
```
@unittest.skip("Not a unit test: this is an integration test that requires setup.")
```